website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -19,6 +19,16 @@ upgrade flow.
 Prior to Consul 1.17.0, audit logs contained timestamps on both the original log file names as well as rotated log file names.
 After Consul 1.17.0, only timestamps will be included in rotated log file names.
 
+#### Service-defaults upstream config (Enterprise)
+Prior to Consul 1.17.0, [Consul-k8s upstreams](/consul/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams)
+that did not explicitly specify a namespace or partition would incorrectly use service-defaults configuration (such as protocol)
+from the default partition / namespace, rather than the local partition / namespace.
```

Comment:
```suggestion
that did not explicitly specify a namespace or partition would incorrectly use service-defaults configurations, such as the incorrect protocol,
from the `default` partition and namespace, rather than the local partition and namespace.
```